### PR TITLE
Product listing page

### PR DESCRIPTION
**Pull request by Ankur9669**
image
Please give some reviews on the page

### CODE REVIEW

**Review by kgajriaAgarwal — APPROVED** [review]
Everything seems  to be great.

**Comment by prankurpandeyy** [comment]
everything looks great to me .

**Review by prankurpandeyy — APPROVED** [review]

**Comment by Sahil25061999** [comment]
No changes required .

File: products.html [hunk 1]
```diff
@@ -0,0 +1,366 @@
...
+          <span class="weight-normal font-small">CLEAR</span>
+        </p>
+        <!--  Sort By Filter Start  -->
+        <div class="filter-type">
+          <p class="weight-semi-bold font-medium">SORT BY</p>
+          <div class="margin-vertical-lg"></div>
+          <ul class="list-style-none filter-list">
+            <li>
+              <input type="radio" name="sort-by" value="price-low-to-high" />
+              <label for="price-high-to-low">Price low to high</label>
```
Comment (Naman-Saxena1) [inline products.html:73]: 1) Price low to high has label for price-high-to-low
2) To Connect Input with label, you have to use id attribute not value
https://stackoverflow.com/a/6293626/13168852

**Review by Naman-Saxena1 — CHANGES_REQUESTED** [review]
I am not sure if gap between sidebar and footer is intentional or not. Everything else looks fine.

File: products.html (hunk 1 above)
Comment (Ankur9669) [reply products.html:73]: Q. I am not sure if gap between sidebar and footer is intentional or not.
Ans: It is intensional

Comment (Ankur9669) [reply products.html:73]: > 1. Price low to high has label for price-high-to-low
> 2. To Connect Input with label, you have to use id attribute not value
>    https://stackoverflow.com/a/6293626/13168852

Ok understood i thought for connecting label with input we need name tag.

**Review by rahulk31 — APPROVED** [review]
Looks good to me Ankur, can be merged. Great work 💯

**Pull request merged**